assets/activation-scripts/activate

```diff
@@ -48,9 +48,10 @@ wait_for_services_socket () {
 
 start_services_blocking () {
   local config_file="$1"; shift
-  local socket_file="$1";
+  local socket_file="$1"; shift
+  local log_dir="$1"
   local log_file
-  log_file="${_FLOX_SERVICES_LOG_FILE:-$(mktemp)}"
+  log_file="$(mktemp --tmpdir="$log_dir" services.XXXXXXXXXX.log)"
```

Comment:
If we were to direct a user to look at the logs, would we just tell them to find the latest in `.flox/logs`? If we use a timestamp instead of a unique key it may help a user narrow down exactly which log file to look in.

Reply:
Good point. I was mostly thinking of it from the perspective of the `flox activate` error, rather than human-to-human. `services.YYYYMMDDHHMMSS.log`, or should we include microseconds to guard against multiple activations in the same second?

Any preference between generating this in `activate.rs` vs using `date`?

Reply:
Done:

> So that users can easily find the latest log in the directory. The
> format includes microseconds in case there are multiple activations
> within the same second, which is plausible when scripting commands.
> 
> I once again considered (and wrote) moving this to Rust but I think it
> makes sense to keep the timestamp calculation as close to the place that
> starts `process-compose` and it means that we don't have to worry about
> leaking additional environment variables into the activation.